### ISSUE TITLE
[CLIN-VAR] Unit Tests for SQON -> ES Mapper

### ISSUE THREAD

**Issue by nyanofthemoon** [cr-ste-justine/clin-project#151]
Please set-up base to run unit tests and write some unit tests for the following SQON to ES mapper features:

- [ ] Generic filter instructions
- [ ] NumericalComparison instructions
- [ ] GenericBoolean filter instructions
- [ ] Composite filter instructions
- [ ] Specific filter instructions
- [ ] "Grouped" filter instructions (query a + query b)

**Comment by nyanofthemoon** [comment]
@vferretti @Emma-Drieux Part 1 nearly completed. Still blocked by missing specs. I created another ticket since I have no idea where the other one is: https://app.zenhub.com/workspaces/clin-sprints-5c59d2c907cc4d6f39e559c2/issues/cr-ste-justine/clin-project/280

**Comment by nyanofthemoon** [comment]
@vferretti I don't see the information for this ticket you mentioned in the meeting this morning. Where can I find this information?

**Comment by vferretti** [comment]
@nyanofthemoon ticket #280

**Comment by nyanofthemoon** [comment]
@vferretti @latch2112 PR https://github.com/cr-ste-justine/clin-proxy-api/pull/105

**Comment by nyanofthemoon** [comment]
"Solo Composite Score" functionality not available: backend work was not completed due to wait for composite UI implementation and we moved to other tasks. Proof here is that it was not listed in "green" in the test matrix:
https://docs.google.com/spreadsheets/d/1RSjmjHFgoJujEeiUL8uvynfci874cOeNJZDzbwcaNrg/edit#gid=0

We need to implement this functionality before we can close this task.
